compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -6788,6 +6788,8 @@ private BType checkInvocationArgs(BLangInvocation iExpr, List<BType> paramTypes,
             }
         }
 
+        List<BRecordType> incRecords = new ArrayList<>();
```

Comment:
Let's use `included` in whatever new construct you're adding.

Reply:
Here we can use `incRecordParams` list instead of creating a new list.

Reply:
This won't work, we need to get the included record params from caller, but `incRecordParams` is created using function signature.